content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -242,6 +242,222 @@ journal.
+
+.. tabs::
+
+   .. tab:: First connection
+
+       #. :guilabel:`The Access Token` should be left empty.
+       #. Click on :guilabel:`Connect`.
+       #. You will be redirected to the Codabox platform where you will have to validate the connection.
+       #. You will be redirected back to Odoo
+       #. The :guilabel:`Access Token` should now be filled, and
+          the :guilabel:`Status` should be set to :guilabel:`Connected`.
+
+       .. image:: belgium/configuration_connection_ok.png
+          :align: center
+          :alt: Configuration of the connection.
+
+       .. important::
+          The :guilabel:`Access Token` should **not be modified**.
+
+   .. tab:: Following connection
+
+       #. Copy the :guilabel:`Access Token` from a previous connection
+       #. Click on :guilabel:`Connect`.
+       #. The Codabox connection linked to the :guilabel:`Access Token`
+          should still exist, therefore you do not need to validate the
+          connection again.
+       #. A new and different :guilabel:`Access Token` will be generated, and the
+          :guilabel:`Status` should be set to :guilabel:`Connected`.
+
+       .. note::
+          By having a different :guilabel:`Access Token` for each client, you can
+          easily revoke the access of a client to your Codabox account without
+          affecting the other clients. One client cannot interfere with another.
+
+Synchronisation
+~~~~~~~~~~~~~~~
+
+Once the connection is established, you are ready to synchronise Odoo
+with Codabox.
+
+.. tabs::
+
+   .. tab:: For CODA files
+
+      CODA files are automatically imported from Codabox every 12 hours. You do
+      not have to do anything. However, if you wish, you can also click on the
+      :guilabel:`Fetch CODA's` button to manually launch the process.
+
+      .. image:: belgium/dashboard_coda.png
+         :align: center
+         :alt: Manually fetch CODA's.
+
+   .. tab:: For SODA files
+
+      You can click on the :guilabel:`Fetch SODA's` button in the Accounting
+      Dashboard to retrieve the latest payroll statements from Codabox (usually
+      once a month).
+
+      .. image:: belgium/dashboard_soda.png
+         :align: center
+         :alt: Manually fetch SODA's.
+
+Revocation
+~~~~~~~~~~
+
+You can revoke the connection between Odoo and Codabox by using the
+:guilabel:`Revoke` button in the Codabox settings which will revoke the
+connection on Odoo's side only.
+
+If you have multiple connections, you will still be able to use the
+Access Token of one of them to easily reconnect Odoo and Codabox for
+another client.
+
+However, if you have only one (remaining) connection, you will not be able
+to use its Access Token to create new connections. In that case, you will
+have to revoke the connection from the Codabox platform too,
+and then recreate a new one following the steps in the
+:ref:`first connection <belgium/codabox-configuration-connection>` section.
+
+Potential issues
+~~~~~~~~~~~~~~~~
```

Comment:
This whole section would be better using this format: 
error message 1
  Explanation

error message 2
  Explanation



Double spacing. It should look like this 
![example-doc](https://github.com/odoo/documentation/assets/95279572/caeefbc3-1624-4940-b126-821226ff38c5)